docs/adrs/license/License_change.MD

```diff
@@ -0,0 +1,100 @@
+# Analysis of the license data structure.
```

Comment:
Pleaes follow ADR convention eg. no need to nest directory structure ... also use the same heading breakdown ... some inspiration here https://adr.github.io/

Reply:
updated, please check again.